tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -1211,7 +1205,6 @@ def test_positive_end_to_end(self, function_entitlement_manifest, target_sat, rh
         rhel_contenthost.install_katello_ca(target_sat)
         # Register client with foreman server using act keys
         rhel_contenthost.register_contenthost(org.label, activation_key_name)
-        assert rhel_contenthost.subscribed
```

Comment:
@pondrejk I thought this would still work. What should be the status here then? We probably should fix this instead of not performing this assertion.

Reply:
I kind of expected it wouldn't, I should have tried though. brought it back, re-running prt